pdsfile/pds4file/rules/cassini_uvis_solarocc_beckerjarmak2023.py

```
@@ -0,0 +1,425 @@
+
+    ARCHIVE_PATHS = archive_paths + pds4file.Pds4File.ARCHIVE_PATHS
+    ARCHIVE_DIRS = archive_dirs + pds4file.Pds4File.ARCHIVE_DIRS
+
+    ############################################################################
+    # DATA_SET_ID is defined as a function rather than a translator
+    ############################################################################
+
+    # Version tables reside in shelves/more_metadata/cassini_uvis_solarocc_beckerjarmak2023 and _0xxx_v1
+    VERSIONS_PATH_AND_KEY = translator.TranslatorByRegex([
+        (r'volumes/COUVIS_0xxx(|_v\d)/(COUVIS_0...)/(.*)/(\w+)\.(DAT|LBL)', 0,
+                    (r'metadata/COUVIS_0xxx/\2/\2\1_versions.tab', r'\4.LBL'))
```

Flake8 (7.2.0)

[error] continuation line over-indented for visual indent

(E127)